packages/server/src/config.ts

```diff
@@ -47,6 +48,8 @@ export class ServerConfig {
 
     const didPlcUrl = process.env.DID_PLC_URL || 'http://localhost:2582'
     const serverDid = process.env.SERVER_DID || ''
+    const recoveryKey = process.env.RECOVERY_KEY || ''
```

Comment:
Is it fair to say this is required?  If so, we might want to eagerly fail if it's missing.

Reply:
yeah that's a good point 👍